GUI/SearchTaskWindow.xaml

```diff
@@ -265,7 +505,17 @@
                                         </TextBlock>
                                     </CheckBox.ToolTip>
                                 </CheckBox>
-                                <CheckBox x:Name="eValueCheckBox" Content="Calculate eValues" />
+                                <CheckBox x:Name="eValueCheckBox" Content="Calculate eValues" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                    <CheckBox.ToolTip>
+                                        <TextBlock>
+                                            This calculates the eValue (the probability of a PSM randomly obtaining its given score) for each PSM and outputs it in the PSM file.
```

Comment:
This calculates the expectation value (the eValue) for each PSM, which is the probability of a PSM obtaining its given score by random chance.

Reply:
There is a significant increase in search time. eValues are not used for FDR and are only calculated for the user's review.